osi_sensorviewconfiguration.proto

```diff
@@ -729,6 +729,51 @@ message CameraSensorViewConfiguration
     //
     repeated WavelengthData wavelength_data = 11;
 
+    // Indicates if and how the the pixel data is ordered
+    //
+    // The default value is PIXEL_ORDER_LEFT_RIGHT_TOP_BOTTOM. 
```

Comment:
@RIFJo I think this field's comment needs some adaptation as well. At least it is referring to a enum value that is no longer there.

Reply:
good catch, fixed in f09484